docs/reference/doofinder.facetquery.term.md

```diff
@@ -0,0 +1,13 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [doofinder](./doofinder.md) &gt; [FacetQuery](./doofinder.facetquery.md) &gt; [term](./doofinder.facetquery.term.md)
+
+## FacetQuery.term property
+
+The facet type. Indicates the type of the facet, one of term or range. will return an error if the facet type is not correct.

```

Comment:
In English, the subject is required so I suggest to add "It" ("It will return an error...")